defaults/main.yml

```diff
@@ -194,5 +194,36 @@ nginx_http_template:
             health_check: max_fails=1 fail_timeout=10s
 
 # Enable creating dynamic templated NGINX stream configuration files.
+# Defaults will not produce a valid configuration. Instead they are meant to showcase
+# the options available for templating. Each key represents a new configuration file.
+# Comment out load_balancer or web_server depending on whether you wish to create a web server
+# or load balancer configuration file.
 nginx_stream_template_enable: false
-nginx_stream_template_listen: 12345
+nginx_stream_template:
+  default:
+    template_file: stream/default.conf.j2
+    conf_file_name: default.conf
+    conf_file_location: /etc/nginx/conf.d/stream/
+    network_stream:
+      streams:
+        default:
+          listen_address: 127.0.0.1
+          listen_port: 443
+          proxy_pass: backend
+          proxy_timeout: 3s
+          proxy_connect_timeout: 1s
+          proxy_protocol: "off"
```

Comment:
```suggestion
          proxy_protocol: false
```

Reply:
I will have to update the logic in the jinja2 template to accept true and false.  I will work on that.

Reply:
Thanks!